### ISSUE TITLE
Noticable lines between pixels when system scale is above 100%

### ISSUE THREAD

**Issue by BenjaminTheHedgefox** [kirjavascript/Flex2#71]
I've been noticing this problem when I zoom in on my current laptop, which has scale set to 125%.

![image](https://github.com/kirjavascript/Flex2/assets/49183041/2c23cdf0-7448-48d0-a59c-a292b3f62c1c)

I'm aware I can lower the scale manually, but that just seems cumbersome... If you can fix this, that'd be wonderful.

**Comment by kirjavascript** [comment]
yeah, been wanting to fix this one for a while, I will see what I can do

**Comment by kirjavascript** [comment]
landed in 1.4.0
